README.md

```diff
@@ -1,2 +1,188 @@
+
+Notes:
+
+- terminating the executable (Ctrl+C) while tailing logs does not stop the run, it just stops tailing logs;
+
+- you cannot confirm or discard the run from the command line. If the deployment ends up in [`UNCONFIRMED`](https://docs.spacelift.io/concepts/run/tracked#unconfirmed) state, the CLI be stuck until the state machine progresses. You will need to go to the run URL and make a decision there;
+
+#### `stack preview` subcommand
+
+```bash
+❯ spacelift-cli stack --id=stack-id preview --help
+NAME:
+   spacelift-cli stack preview - Start a preview (proposed run)
+
+USAGE:
+   spacelift-cli stack preview [command options] [arguments...]
+
+CATEGORY:
+   Run management
+
+OPTIONS:
+   --sha value  Commit SHA for the newly created run
+   --tail       Indicate whether to tail the run (default: false)
+   --help, -h   show help (default: false)
+```
+
+This subcommand creates a [proposed run](https://docs.spacelift.io/concepts/run/proposed) on a stack. By default, it uses the latest tracked commit, but you can also point it to an arbitrary SHA using the `--sha` flag. Also by default, this command only creates a run and prints out the URL where the run can be accessed. If the `--tail` flag is set however, run logs will be retrieved until the run terminates. If the run is tailed, the exit code of the command will depend on the outcome of the run - if the run is `FINISHED`, the command succeeds (exits with `0`), otherwise it fails.
+
+#### `stack logs` subcommand
+
+```bash
+❯ spacelift-cli stack --id=stack-id logs --help
+NAME:
+   spacelift-cli stack logs - Show logs for a particular run
+
+USAGE:
+   spacelift-cli stack logs [command options] [arguments...]
+
+CATEGORY:
+   Run management
+
+OPTIONS:
+   --run value  ID of the run
+   --help, -h   show help (default: false)
+```
+
+This subcommand prints out (and tails) logs for an existing run or task. The exit code of the command will depend on the outcome of the run - if the run is `FINISHED`, the command succeeds (exits with `0`), otherwise it fails.
+
+Note that `run` is a required parameter here and represents the unique run ID, one that looks like `01F2KB8SARWF3V2PSFYXK5D0S7`.
+
+#### `stack task` subcommand
+
+```bash
+❯ spacelift-cli stack --id=stack-id task --help
+NAME:
+   spacelift-cli stack task - Perform a task in a workspace
+
+USAGE:
+   spacelift-cli stack task [command options] [arguments...]
+
+CATEGORY:
+   Run management
+
+OPTIONS:
+   --noinit    Indicate whether to skip initialization for a task (default: false)
+   --tail      Indicate whether to tail the run (default: false)
+   --help, -h  show help (default: false)
+```
+
+This subcommand starts a [task](https://docs.spacelift.io/concepts/run/task) against a stack. The command for the task itself goes after the CLI bit. It can be quoted to prevent any confusion in from shell tokenization. Example:
```

Comment:
```suggestion
This subcommand starts a [task](https://docs.spacelift.io/concepts/run/task) against a stack. The command for the task itself must be specified after all the other `spacelift-cli` command arguments. It can be quoted to prevent any confusion from shell tokenization. Example:
```